pkg/tsdbctl/tsdbctl.go

```diff
@@ -191,6 +191,8 @@ func (rc *RootCommandeer) populateConfig(cfg *config.V3ioConfig) error {
 	}
 	if rc.logLevel != "" {
 		cfg.LogLevel = rc.logLevel
+	} else {
+		cfg.LogLevel = "info"
```

Comment:
Changing the default logging level from `debug` to `info` should include updating the `DefaultLoggingLevel` const in *pkg/config/config.go* and using this const in relevant locations such as in this `*tsdbctl.go* line. (We already use this in this file, *tsdbctl.go*, for the CLI help text of the global `-v|--log-level` flag.)

Reply:
Not exactly. We use to set log level to `debug` by default for `-v` option without args in CLI.
If used didn't use `-v` the log level should be `info` (according to @gshatz )

Reply:
the default of "info" is only for the CLI - any other apps using TSDB should maintain their defaults or get the nuclio-logger default of "warn"